### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by kimitaWanjohi**
In this PR I did:

- Add person class.
- Add a student class that extends the person class.
- Add a teacher class that extends the person class

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    false
+  end
+
+  private
+
+  def of_age?
+    return true if age >= 18
+
+    false
+  end
```
Comment (levy002) [inline person.rb:24]: - [OPTIONAL] You can simplify the `of_age?` method by using a single comparison `age >= 18` to determine if a student is of age. These simplifications make the code cleaner and more concise while retaining its functionality."
```suggestion
  def of_age?
   age >= 18
  end
```

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission
+
+    false
+  end
```
Comment (levy002) [inline person.rb:16]: - [OPTIONAL] When working on the `can_use_services?` method, you can simplify the code by directly returning the result of the condition `is_of_age? || @parent_permission`. This takes advantage of the fact that the condition itself evaluates to either `true` or `false`, which matches the desired return value. These simplifications make the code cleaner and more concise while retaining its functionality.

```suggestion
  def can_use_services?
     is_of_age? || @parent_permiss
  end
```

**Review by levy002 — APPROVED** [review]
## Hello @kimitaWanjohi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
- No linter error :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (kimitaWanjohi) [reply person.rb:24]: done

**Pull request merged**